frontend/src/queries.js

```diff
@@ -1349,8 +1349,6 @@ export const createMatchStatsEvent = async ({ match_id, body }) => {
 
   const data = await response.json();
 
-  await delay(800);
-
```

Comment:
this might've caused some latency too :grimacing: :grimacing:

Reply:
lol